src/app/fdctl/config/default.toml

```diff
@@ -1,585 +1,663 @@
-# configuration determines how many of each tile to place on which CPU core to
-# create a functioning system, below is the individual settings that can change
-# behavior of the tiles.
+# Tiles are described in detail in the layout section above.  While the
+# layout configuration determines how many of each tile to place on
+# which CPU core to create a functioning system, below is the individual
+# settings that can change behavior of the tiles.
 [tiles]
-    # A networking tile is responsible for sending and receiving packets on the
-    # the network.  Each networking tile is bound to a specific queue on a
-    # network device.  For example, if you have one network device with four
-    # queues, you must run four net tiles.
+    # A networking tile is responsible for sending and receiving packets
+    # on the the network.  Each networking tile is bound to a specific
+    # queue on a network device.  For example, if you have one network
+    # device with four queues, you must run four net tiles.
     #
-    # Net tiles will multiplex in both directions, fanning out packets to
-    # multiple parts of Firedancer that can receive and handled them, like QUIC
-    # tiles and the Turbine retransmission engine.  Then also fanning in from
-    # these various network senders to transmit on the queues we have available.
+    # Net tiles will multiplex in both directions, fanning out packets
+    # to multiple parts of Firedancer that can receive and handled them,
+    # like QUIC tiles and the Turbine retransmission engine.  Then also
+    # fanning in from these various network senders to transmit on the
+    # queues we have available.
     [tiles.net]
-        # Which interface to bind to for network traffic. Currently only one
-        # interface is supported for networking.  If this is empty, the default
-        # is interface used to route to 8.8.8.8, you can check what this is with
-        # `ip route get 8.8.8.8`
+        # Which interface to bind to for network traffic. Currently only
+        # one interface is supported for networking.  If this is empty,
+        # the default is the interface used to route to 8.8.8.8, you can
+        # check what this is with `ip route get 8.8.8.8`
         #
-        # If developing under a network namespace with [netns] enabled, this
-        # should be the same as [development.netns.interface0].
+        # If developing under a network namespace with [netns] enabled,
+        # this should be the same as [development.netns.interface0].
         interface = ""
 
-        # Firedancer uses XDP for fast packet processing. XDP supports two
-        # modes, XDP_SKB and XDP_DRV. XDP_DRV is preferred as it is faster,
-        # but is not supported by all drivers.
+        # Firedancer uses XDP for fast packet processing. XDP supports
+        # two modes, XDP_SKB and XDP_DRV. XDP_DRV is preferred as it is
+        # faster, but is not supported by all drivers.
```

Comment:
Add something like "this argument should be either the string `skb` or the string `drv`"